communication/mavsdkvehicleserver.cpp

```diff
@@ -32,6 +35,9 @@ MavsdkVehicleServer::MavsdkVehicleServer(QSharedPointer<VehicleState> vehicleSta
     mParamServer->provide_param_int("CAL_MAG0_ID", 1);
```

Comment:
Half question/half comment: currently MavsdkVehicleServer-internal parameters are advertised only. All classes that want to advertise changeable parameters would need to get access to mParamServer, right? I think in that case we will need to wrap it in another class to make it more general/independent of MAVSDK (otherwise all classes advertising parameters would become MAVSDK-dependent, e.g., WaypointFollower advertising some radius) but this is stuff for another PR.

Reply:
I agree